docker-compose.local.yml

```diff
@@ -106,6 +106,8 @@ kpi:
     # Dev: Use live source directories from the host machine.
     # - ../kpi:/srv/src/kpi
     #- ../formpack/src/formpack:/usr/local/lib/python2.7/dist-packages/formpack
+    # In case you're developping locally and solely using local storage uncomment the following line so you can access media uploads locally within KPI
+    # - ./.vols/kobocat_media_uploads:/srv/src/kobocat/media
```

Comment:
@yahyazini, this isn't just for local development: we have some (security conscious) organizations who refuse to use services like S3 and store all their media locally. Could you enable the volume mapping by default and also add it to `docker-compose.server.yml`?